src/Teams/beta/custom/SetMgBetaTeamRscConfiguration_Update.cs

```diff
@@ -0,0 +1,447 @@
+﻿namespace Microsoft.Graph.Beta.PowerShell.Cmdlets
+{
+    using Microsoft.Graph.Beta.PowerShell.Models;
```

Comment:
Let's update public contract to:

Get-MgBetaTeamRscConfiguration

Get-MgBetaChatRscConfiguration

Set-MgBetaTeamRscConfiguration -State {DisabledForAllApps | EnabledForPreApprovedAppsOnly | EnabledForAllApps}

Set-MgBetaTeamChatConfiguration -State {DisabledForAllApps | EnabledForPreApprovedAppsOnly | EnabledForAllApps